lux/executor/PandasExecutor.py

```diff
@@ -515,6 +515,7 @@ def compute_stats(self, ldf: LuxDataFrame):
         ldf.unique_values = {}
         ldf._min_max = {}
         ldf.cardinality = {}
+        ldf.length = len(ldf)
 
```

Comment:
Ahh I understand that `ldf.length` now exists, but it doesn't always exist. Can we set this upon initialization?

Reply:
@dorisjlee So currently the `ldf.length` parameter is being updated whenever `compute_stats` is called by maintain metadata. Whenever a dataframe is manipulated in some way, this parameter is adjusted alongside all of the other parameters like `_min_max`, `unique_values`, etc.

Reply:
It should be, but we would need to make sure that it doesn't break anything. Ideally, we should [override the `__len__` variable](https://stackoverflow.com/questions/30483752/is-it-possible-to-modify-the-behavior-of-len) inside the `LuxSQLTable` to change what we get from len(tbl). That way we would not have to change the behavior of `PandasExecutor` or `LuxDataFrame`, which has more potential for causing errors down the line.

Reply:
Oh yeah overriding `__len__` makes a lot of sense since we could keep the same syntax for the `LuxSQLTable` and `LuxDataFrame` objects. I'll look into trying that out